src/monorepo-utils/src/Git.js

```diff
@@ -52,8 +52,13 @@ const Git = {
   },
 
   getChangedFiles() /*: $ReadOnlyArray<string> */ {
-    const rawChanges = git('diff', '--name-only', 'origin/master...HEAD');
-    return __parseRows(rawChanges);
+    try {
+      const rawChanges = git('diff', '--name-only', 'origin/master...HEAD');
+      return __parseRows(rawChanges);
+    } catch {
+      // This command fails if you have no remote, empty string should be a reasonable fallback
```

Comment:
Would it make more sense to have `origin/master` configurable (so you can set just `master`) instead of silently ignoring the missing ref?

Reply:
🚀